Singer.API/ClientApp/src/app/modules/legalguardians/services/careRegistrationResult.dto.ts

```diff
@@ -0,0 +1,3 @@
+export interface CareRegistrationResultDTO {
```

Comment:
What's this DTO doing here? It looks like it lost its mommy, please explain it how to find its way to the DTOs folder ;)